src/handlers/privateResolver.ts

```diff
@@ -25,6 +25,20 @@ import { LocalAuthorityRegisterRequestsRepository } from '../repository/localAut
 import { SchoolDataRepository } from '../repository/schoolDataRepository';
```

Comment:
Great work, can we also replicate this for the public resolver please?

Reply:
Thanks, I've now done the public resolver too